_layouts/default.html

```diff
@@ -33,10 +33,12 @@
             </div>
             <br>
             <div class="menu">
-        	<a href="index"><img src="assets/images/home.png" height="15px"></a>
+        	<a href="index"><img src="/assets/images/home.png" height="15px"></a>
+                <a href="https://petstore.swagger.io/?url=https://raw.githubusercontent.com/Materials-Consortia/optimade-python-tools/master/openapi/openapi.json" target="_blank">API Documentation
```

Comment:
```suggestion
                <a href="https://petstore.swagger.io/?url=https://petstore.swagger.io/?url=https://raw.githubusercontent.com/Materials-Consortia/OPTiMaDe/v1.0.0-rc.1/schemas/openapi_schema.json" target="_blank">API Documentation
```

Reply:
Thanks @CasperWA , I fully agree.
I wasn't aware that the spec repository also contained an openapi.json.

Reply:
Sure - it's a direct copy of the one created in `optimade-python-tools` at the moment, but we are working on moving the generation either to a separate repo or into the spec repo.